docs/elements/faq.html

```diff
@@ -0,0 +1,47 @@
+<article-header>FAQ</article-header>
+<article-paragraphs>
+    <p>Here are some of the most frequently asked questions about using the BLT and their answers.</p>
+</article-paragraphs>
+
+<article-paragraphs>
+    <p><strong>Q: What is this package?</strong></p>
+    <p>A: BLT, Behat + Laravel Toolbox, is a toolkit designed to facilitate the testing of Laravel applications using
+        Behat. It provides many features and traits that simplify the integration and testing process, making it easier
+        to write and maintain robust tests.</p>
+</article-paragraphs>
+<article-paragraphs>
+    <p><strong>Q: How do I install BLT?</strong></p>
+    <p>A: You can install BLT via Composer by running the command:</p>
+    <p><code>composer require blumilksoftware/blt</code></p>
+</article-paragraphs>
+<article-paragraphs>
+    <p><strong>Q: How do I configure BLT?</strong></p>
+    <p>A: BLT configuration can be customized in the <code>config/blt.php</code> file. You can set namespaces,
+        endpoints, and other settings as per your requirements.</p>
+</article-paragraphs>
+
+<article-paragraphs>
+    <p><strong>Q: How do I mock services in my tests?</strong></p>
+    <p>A: You can use the Application trait to mock services. It allows you to replace actual services with mock
+        implementations for isolated testing.</p>
+</article-paragraphs>
+
+<article-paragraphs>
+    <p><strong>Q: I have a question, how can I contact you?</strong></p>
+    <p>A: You can contact us interns directly or write to the contact page at Blumilk.</p>
+    <p>Emails:</p>
+    <p>jakub.kermes@studenci.collegiumwitelona.pl</p>
+    <p>piotr.fedak@studenci.collegiumwitelona.pl</p>
+    <p>Website: <a href="https://www.blumilk.pl/contact" target="_blank">https://www.blumilk.pl/contact</a></p>
+</article-paragraphs>
+
+<article-paragraphs>
+    <p><strong>Q: What does BLT stand for?</strong></p>
+    <p>A: BLT is a humorous acronym referring to the classic sandwich, Bacon, Lettuce, and Tomato. However, in our
+        context, it represents "Behat + Laravel Toolbox," a toolkit for testing Laravel applications with Behat.</p>
+</article-paragraphs>
+
+<article-paragraphs>
+    <p><strong>Q: I found a bug, can I report it?</strong></p>
+    <p>A: Of course, please contact us or you can write an issue for that.</p>
```

Comment:
But where the user can write a bug? On GitHub? By e-mail?

Reply:
Okey, i will add a github link and some emails